test/e2e/all_in_one_test.go

```diff
@@ -44,9 +44,10 @@ func TestDeployAllInOneDBLESSLegacy(t *testing.T) {
 	ctx, env := setupE2ETest(t)
 
 	t.Log("deploying kong components")
-	manifest := getTestManifest(t, "../../deploy/single/all-in-one-dbless-legacy.yaml")
-	deployment := deployKong(ctx, t, env, manifest)
-
+	const manifestPath = "../../deploy/single/all-in-one-dbless-legacy.yaml"
```

Comment:
nit: For readability I'd personally move this to be the fist (`const`) block in this func.